src/common/src/util/epoch.rs

```diff
@@ -116,8 +116,11 @@ impl Epoch {
 
 pub const EPOCH_AVAILABLE_BITS: u64 = 16;
 pub const MAX_SPILL_TIMES: u16 = ((1 << EPOCH_AVAILABLE_BITS) - 1) as u16;
-pub const EPOCH_MASK: u64 = (1 << EPOCH_AVAILABLE_BITS) - 1;
-pub const MAX_EPOCH: u64 = u64::MAX & !EPOCH_MASK;
+// Low EPOCH_AVAILABLE_BITS bits set to 1
+pub const EPOCH_SPILL_TIME_MASK: u64 = (1 << EPOCH_AVAILABLE_BITS) - 1;
+// High (64-EPOCH_AVAILABLE_BITS) bits set to 1
+pub const EPOCH_MASK: u64 = !EPOCH_SPILL_TIME_MASK;
```

Comment:
The `EPOCH_MASK` is fully flipped compared to before this PR. It is used anywhere else? If not I think we can remove this const or make it private.